website/documentation/contribute/_index.md

```diff
@@ -65,7 +65,7 @@ We kindly ask you to follow the [Pull Request Checklist](#pull-request-checklist
 
 * Commits should be as small as possible, while ensuring that each commit is correct independently (i.e., each commit should compile and pass tests).
 
-* Test your changes as thoroughly as possible before your commit them. Preferably, automate your test by unit / integration (e.g. [Gardener integration testing](https://github.com/gardener/gardener/blob/master/docs/testing/integration_tests.md)) tests. If tested manually, provide information about the test scope in the PR description (e.g. “Test passed: Upgrade K8s version from 1.14.5 to 1.15.2 on AWS, Azure, GCP, Alicloud, Openstack.”).
+* Test your changes as thoroughly as possible before your commit them. Preferably, automate your test by unit / integration (e.g. [Gardener integration testing](https://github.com/gardener/gardener/blob/master/docs/development/testing.md#integration-tests-envtests)) tests. If tested manually, provide information about the test scope in the PR description (e.g. “Test passed: Upgrade K8s version from 1.14.5 to 1.15.2 on AWS, Azure, GCP, Alicloud, Openstack.”).
```

Comment:
as we talk about both unit and integration tests, shall we use https://github.com/gardener/gardener/blob/master/docs/development/testing.md instead of https://github.com/gardener/gardener/blob/master/docs/development/testing.md#integration-tests-envtests ?